src/components/RelatedVideoList/RelatedVideos/RelatedVideos.spec.jsx

```diff
@@ -105,16 +112,21 @@ describe('Testing VideoList', () => {
         },
       ];
 
-      useVideos.mockReturnValueOnce([returnedArray, false]);
-
-      const { container } = render(<App />);
+      useVideos.mockReturnValueOnce({ videoList: returnedArray, isLoading: false });
 
-      expect(container).toHaveTextContent(
-        /Wizeline transforms how teams build technology/i
-      );
-      expect(container).toHaveTextContent(
-        /Follow Hector Padilla, Wizeline Director of Engineering/i
+      const { getByText } = render(
+        <BrowserRouter>
+          <DarkProvider>
+            <ThemesProvider>
+              <RelatedVideos relatedToVideoId={relatedToVideoId} />
+            </ThemesProvider>
+          </DarkProvider>
+        </BrowserRouter>
       );
+
+      expect(useVideos).toBeCalledWith(relatedToVideoId, REQ_TYPE);
+      expect(getByText(/Wizeline title1/i)).toBeInTheDocument();
+      expect(getByText(/Video Tour | Welcome /i)).toBeInTheDocument();
```

Comment:
Nice tests. I like that you mocked your hook results, this way you have control over what gets rendered in your tests 👍🏽 .